UnsplashPhotoPicker/UnsplashPhotoPicker/Classes/Controllers/UnsplashPhotoPickerViewController.swift

```diff
@@ -159,22 +158,13 @@ class UnsplashPhotoPickerViewController: UIViewController {
         NotificationCenter.default.addObserver(self, selector: #selector(keyboardWillHideNotification(_:)), name: UIResponder.keyboardWillHideNotification, object: nil)
     }
 
-    private func setupNavigationBar() {
-        updateTitle()
-        navigationItem.leftBarButtonItem = cancelBarButtonItem
-
-        if Configuration.shared.allowsMultipleSelection {
-            doneBarButtonItem.isEnabled = false
-            navigationItem.rightBarButtonItem = doneBarButtonItem
-        }
-    }
-
```

Comment:
Please let's try to reduce the number of changes... can we leave this method as is?